### PR TITLE
(enhc) remove bill amount item on patients table and prevent page redirect after successful bill payment

### PR DESCRIPTION
**Pull request by NjokiCynthia**
…fter successful bill payment

## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-billing-app/src/invoice/payments/payment-history/payment-history.component.tsx [hunk 1]
```diff
@@ -14,10 +14,10 @@ const PaymentHistory: React.FC<PaymentHistoryProps> = ({ bill }) => {
       key: 'dateCreated',
       header: 'Date of payment',
     },
-    {
-      key: 'amount',
-      header: 'Bill amount',
-    },
+    // {
```
Comment (ojwanganto) [inline packages/esm-billing-app/src/invoice/payments/payment-history/payment-history.component.tsx:17]: You can just remove the commented lines

File: packages/esm-billing-app/src/invoice/payments/payments.component.tsx [hunk 2]
```diff
@@ -62,7 +62,7 @@ const Payments: React.FC<PaymentProps> = ({ bill, selectedLineItems }) => {
           kind: 'success',
           timeoutInMs: 3000,
         });
-        handleNavigateToBillingDashboard();
+        // handleNavigateToBillingDashboard();
```
Comment (ojwanganto) [inline packages/esm-billing-app/src/invoice/payments/payments.component.tsx:65]: We can also remove this

**Comment by NjokiCynthia** [comment]
https://github.com/palladiumkenya/kenyaemr-esm-3.x/assets/45117658/e824c65e-6346-48e2-9277-5e2397ac2827

**Comment by NjokiCynthia** [comment]
row_edit

**Review by ojwanganto — APPROVED** [review]
Thanks, @NjokiCynthia

**Review by donaldkibet — APPROVED** [review]
Thanks @NjokiCynthia LGTM 🤝

**Pull request merged**